dhis-2/dhis-services/dhis-service-dxf2/src/main/java/org/hisp/dhis/dxf2/datavalueset/DefaultDataValueSetService.java

```diff
@@ -933,7 +933,7 @@ private void saveDataValueDelete(
                 internalValue,
                 existingValue.getValue(),
                 context.getStoredBy(dataValue),
-                ChangeLogType.DELETE);
+                TrackerAuditType.DELETE);
```

Comment:
Here is the same, `DataValueSet` is not part of tracker so it seems a stretch that audit is prefixed with tracker